client/src/App.js

```diff
@@ -1,13 +1,41 @@
-import React from "react";
+import React, { Component } from 'react';
 import "./App.css";
-import CreateTimeTable from "./components/CreateTimeTable.jsx";
+import Navbar from "./components/Navbar";
+import Dashboard from "./components/Dashboard"
+import CreateTimeTable from "./components/CreateTimeTable"
+import ShareTimeTable from "./components/ShareTimeTable"
+import AboutUs from "./components/AboutUs"
 
-function App() {
-  return (
-    <div className="App">
-      <CreateTimeTable />
-    </div>
-  );
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { currentTab: "Dashboard" }
+    this.changeTab=this.changeTab.bind(this);
+  }
+  changeTab(input)
+  {
+    console.log(input.target.text);
+    let selectedTab=input.target.text;
+    this.setState({
+      currentTab: selectedTab
+    })
+  }
+  render() { 
+    return ( <div>
+      <Navbar action={this.changeTab}/>
+      <div>
+        {this.state.currentTab==="Dashboard"?
+          <Dashboard />
+          :this.state.currentTab==="Create TimeTable" ?
+          <CreateTimeTable />
+          :this.state.currentTab==="Share TimeTable" ?
+          <ShareTimeTable />
+          :this.state.currentTab==="About Us"?
+          <AboutUs />:<h1 align="center">"Page not available"</h1>
+        }
+      </div>
+    </div> );
```

Comment:
This whole thing about the navbar. Do we want to switch views like this, or do it the React way, using routers?

Also this part of the logic is more suited within `navbar.js` imo, change the `hrefs` there.